src/emulator/functionsEmulator.ts

```diff
@@ -195,7 +182,12 @@ export class FunctionsEmulator implements EmulatorInstance {
       const runtimeReq = http.request(
         {
           method,
-          path: req.url, // 'url' includes the query params
+          path:
+            "/" +
+            req.url
+              .split("/")
+              .slice(4)
```

Comment:
Magic things like `.slice(4)` make me uncomfortable.  Is there some way to do this with a regex or make it more clear what you're extracting here?

Reply:
SLICING MAKES YOU UNCOMFORTABLE BUT YOU WANT A REGEX

![WHAT?](https://jpg.cool/what.gif)

Nah but seriously, we could put it into a separate functions with some tests if it felt less magical

Reply:
It's the "4" that I don't like.  It's not a named section (which a regex could have).